oeis.py

```diff
@@ -402,6 +402,17 @@ def A000108(start: int = 0, limit: int = 20) -> Collection[int]:
     return sequence
 
 
+@oeis
+def A007953(start: int = 0, limit: int = 20) -> Collection[int]:
+    "Digital sum (i.e., sum of digits) of n; also called digsum(n)."
+    sequence = []
+
+    for n in range(start, start + limit):
+        sequence.append(sum(int(d) for d in str(n)))
+
+    return sequence
```

Comment:
Why not using a list comprehension, as in:

```python
def A007953(start: int = 0, limit: int = 20) -> Collection[int]:
    return [sum(int(d) for d in str(n)) for n in range(start, start+limit]
```

?

Reply:
I just didn't think of it, I'm not used to this syntax, but I can update the sequence.

Reply:
Though you were used to the syntax as you used it: `sum(int(d) for d in str(n))`.

I typically don't recommand list comprehension inside lists comprehensions for readability reasons, but if you find it readable it's OK, if you don't find it readable, maybe keep it as is.

Reply:
I used it here to try because I've seen others use it, and since it works, I left it as it is, but I don't find it readable.

Reply:
Single list comprehensions gets more readable than not using them but it takes time to accustomate. Lists comprehensions in lists comprehensions are typically hard to read and modify.

Compare using 0 list comprehension to using one to using two:

## Using 0 list comprehension

I find it too bloaty, it's hard to read:

```python
    sequence = []

    for n in range(start, start + limit):
        acc = 0
        for d in str(n):
             acc += int(d)
        sequence.append(acc)

    return sequence
```

## Using 1 list comprehension

The structure is easier to understand, density is OK:

```python
    sequence = []

    for n in range(start, start + limit):
        sequence.append(sum(int(d) for d in str(n)))

    return sequence
```

## Using 2 lists comprehensions

Having list comprehension inside a list comprehension is typically hard to read and to avoid. I find this one almost OK, but maybe because I'm abustomed to them a bit too much.

```python
    return [sum(int(d) for d in str(n)) for n in range(start, start+limit]
```